packages/indexing/src/local-index-api.ts

```diff
@@ -6,22 +6,28 @@ import type {
   PaginationQuery,
   StreamState,
   DiagnosticsLogger,
+  LoadOpts,
+  Stream,
 } from '@ceramicnetwork/common'
 import type { DatabaseIndexApi, IndexModelArgs } from './database-index-api.js'
-import type { Repository } from '../state-management/repository.js'
 import { IndexStreamArgs } from './database-index-api.js'
-import { StreamID } from '@ceramicnetwork/streamid'
+import { type CommitID, StreamID } from '@ceramicnetwork/streamid'
 import { IndexingConfig } from './build-indexing.js'
-import { makeIndexApi } from '../initialization/make-index-api.js'
+import { makeIndexApi } from './make-index-api.js'
 import { Networks } from '@ceramicnetwork/common'
 import { Model } from '@ceramicnetwork/stream-model'
-import { ISyncQueryApi } from '../history-sync/interfaces.js'
+import { ISyncQueryApi } from './history-sync/interfaces.js'
+
+export type CeramicCoreApi = {
```

Comment:
a comment on this type would be helpful to make clear why we need this instead of being able to just use `CeramicApi`

Reply:
Yes good call, I'll add this when I move this type, probably to the common package